README.md

```diff
@@ -0,0 +1,130 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  
+  <br/>
+
+  <h3><b>Welcome...</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [To Do list] <a name="about-project"></a>
+
+To Do List.
+
+**[To Do List]** A simple HTML list of To Do tasks
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+ 1. HTML5
+ 2. CSS3
+ 3. JavaScript
+
+
+### Key Features <a name="key-features"></a>
+
+  1. List Structure
+  2. Dynamically created list of tasks.
+  3. set style rules for To Do list
+  
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+install node and vscode
+```
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone git@github.com:MargaretTechworld/ToDo-Tasks.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd ToDo-Tasks
+  npm install
+```
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+npm start
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+👤 **MargaretTechworld**
+
+- GitHub: [MargaretTechworld](https://github.com/MargaretTechworld)
+- Twitter: [KojoMargaret](https://twitter.com/KojoMargaret)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+1. Add functionality to add and remove task
+2. Make list interactive
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+##👍 Contributors
+Feel free to check the [issues page](https://github.com/MargaretTechworldToDo-tasks/issues).
```

Comment:
- [ ] Please make sure that all the links in your README file are working! Currently, this link is broken. 🤓 🔧